docs/releases.md

```diff
@@ -1,6 +1,54 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release  v0.0.1-2678 (Latest)
+## Release  v0.0.1-2727 (Latest)
+> 24th March, 2022
+
+#### About
+
+This build implements received(rx) rate statistics of port metrics. It also has auto destination mac learning support using choice as `auto` in ethernet destination mac of packet header.
```

Comment:
we can avoid duplication of information - addition of comment to refer to the "New features" section for added functionalities should suffice.